pages/auth/login/index.tsx

```diff
@@ -1,98 +1,108 @@
-                    .then(userResponse => {
-                        const loggedInUser = userResponse.data.user
-                        if (loggedInUser) {
-                            router.push('/')
-                        }
-                    })
-            }
-        )
-
-    }, [router])
-
-    const signInWithGoogle = async () => {
-        authService.signInWithGoogle()
-            .then(resp => console.log(resp))
-    }
 
-    return (
-        <div className={containerClassName}>
-            <div className="flex flex-column align-items-center justify-content-center">
-                <img src={`${contextPath}/layout/images/logo-${layoutConfig.colorScheme === 'light' ? 'dark' : 'white'}.svg`} alt="Seattle Humane logo" className="mb-5 w-6rem flex-shrink-0" />
-                <div style={{ borderRadius: '56px', padding: '0.3rem', background: 'linear-gradient(180deg, var(--primary-color) 10%, rgba(33, 150, 243, 0) 30%)' }}>
-                    <div className="w-full surface-card py-8 px-5 sm:px-8" style={{ borderRadius: '53px' }}>
-                        <div className="text-center mb-5">
-                            <img src={`${contextPath}/demo/images/login/avatar.png`} alt="Image" height="50" className="mb-3" />
-                            <div className="text-900 text-3xl font-medium mb-3">Welcome, Isabel!</div>
-                            <span className="text-600 font-medium">Sign in to continue</span>
-                        </div>
-
-                        <div>
-                            <label htmlFor="email1" className="block text-900 text-xl font-medium mb-2">
-                                Email
-                            </label>
-                            <InputText id="email1" type="text" placeholder="Email address" className="w-full md:w-30rem mb-5" style={{ padding: '1rem' }} />
+const LoginPage = () => {
+  const { layoutConfig } = useContext(LayoutContext);
+  const contextPath = getConfig().publicRuntimeConfig.contextPath;
+  const router = useRouter();
+  const containerClassName = classNames(
+    "surface-ground flex align-items-center justify-content-center min-h-screen min-w-screen overflow-hidden",
+    { "p-input-filled": layoutConfig.inputStyle === "filled" }
+  );
 
-                            <label htmlFor="password1" className="block text-900 font-medium text-xl mb-2">
-                                Password
-                            </label>
-                            <Password id="password1" value={password} onChange={(e) => setPassword(e.target.value)} placeholder="Password" toggleMask className="w-full mb-5" inputClassName="w-full p-3 md:w-30rem"></Password>
+  useEffect(() => {
+    supabaseClient.auth.onAuthStateChange(async (event, session) => {
+      supabaseClient.auth.getUser().then((userResponse) => {
+        const loggedInUser = userResponse.data.user;
+        if (loggedInUser) {
+          router.push("/");
+        }
+      });
+    });
+  }, [router]);
 
```

Comment:
This is ok for now. On next PR let's abstract the supabaseClient stuff into authService.

Reply:
Will do